python/lsst/dc2gen3/dc2gen3.py

```diff
@@ -127,10 +129,8 @@ def load(cls, param_fname):
         task_config.load(params['convertRepo_config'])
         init_kwargs['convert_repo_config'] = task_config
 
-        init_kwargs['calibs'] = {}
-        for key, value in params['calibs'].items():
-            collection_name = instrument.makeCollectionName(value)
-            init_kwargs['calibs'][key] = collection_name
+        init_kwargs['calibs'] = \
+            [CalibRepo(**kws) for kws in params['calibs']]
```

Comment:
LSST code uses kwds instead of kws for "keywords" so would be easier to read if that was done here.